packages/vscode-extension/src/utilities/expoCli.ts

```diff
@@ -20,20 +20,35 @@ export function shouldUseExpoCLI() {
 
   const appRootFolder = getAppRootFolder();
   let hasExpoCLIInstalled = false,
-    hasExpoCommandsInScripts = false;
+    hasExpoCommandsInScripts = false,
+    hasExpoConfigInAppJson = false,
+    hasExpoConfigInAppConfigJs = false;
   try {
     hasExpoCLIInstalled =
       require.resolve("@expo/cli/build/src/start/index", {
         paths: [appRootFolder],
       }) !== undefined;
   } catch (e) {}
 
+  try {
+    const appJson = require(path.join(appRootFolder, "app.json"));
```

Comment:
should we use `requireNoCache` here? seems like we just want to read the contents of the file